services/blockchain-indexer/shared/dataService/business/transactions.js

```diff
@@ -174,26 +154,28 @@ const getTransactions = async params => {
 			const indexedTxInfo = resultSet.find(txInfo => txInfo.id === transaction.id);
 
 			const senderAccount = await getIndexedAccountInfo(
-				{ address: transaction.senderAddress, limit: 1 },
-				['address', 'publicKey', 'name'],
+				{ address: getBase32AddressFromPublicKey(transaction.senderPublicKey), limit: 1 },
```

Comment:
Instead of calculating the address twice, can we please compute it once and store it in a local variable?
`getBase32AddressFromPublicKey(transaction.senderPublicKey)`